Doc/library/asyncio-eventloop.rst

```diff
@@ -980,6 +980,18 @@ Handle
       .. versionadded:: 3.7
 
 
+SendfileNotAvailable
+--------------------
+
+
+.. exception:: SendfileNotAvailable
```

Comment:
One more nit: can we call it `SendfileNotAvailableError`? The `Error` suffix is used for 99% of exceptions in Python.

Reply:
Good catch. Done